lib/extension/list.extension.js

```diff
@@ -0,0 +1,40 @@
+'use strict'; // eslint-disable-line strict
+/**
+ *  Class of an extension doing the simple listing
+ */
+class List {
+    /**
+     *  Constructor
+     *  Set the logger and the res
+     *  @param {Object} parameters - The parameters you sent to DBD
+     *  @param {RequestLogger} logger - The logger of the request
+     *  @return {undefined}
+     */
+    constructor(parameters, logger) {
+        this.logger = logger;
+        this.res = [];
+    }
+
+    /**
+     *  Function apply on each element
+     *  Just add it to the array
+     *  @param {Object} elem - The data from the database
+     *  @return {Boolean} - True = continue the stream
+     */
+    filter(elem) {
```

Comment:
Yes, but the filter function is the function called on each element given by the readStream on the database it's the same function for ALL extensions, it's a convention.